server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/json/Serializer.scala

```diff
@@ -421,9 +426,13 @@ class Serializer @Inject() (mailboxIdFactory: MailboxId.Factory) {
 
   def serialize(errors: JsError): JsValue = Json.toJson(errors)
 
-  def serialize(vacationResponse: VacationResponse): JsValue = Json.toJson(vacationResponse)
+  def serialize(vacationResponse: VacationResponse)(implicit vacationResponseWrites: Writes[VacationResponse]): JsValue = Json.toJson(vacationResponse)
+
+  def serialize(vacationResponseGetResponse: VacationResponseGetResponse)(implicit vacationResponseWrites: Writes[VacationResponse]): JsValue =
```

Comment:
Do we use `serialize(vacationResponseGetResponse)` without specifying the vacationResponseWrites part of the arguments? If not we might very well ommit it and just keep `def serialize(vacationResponseGetResponse: VacationResponseGetResponse, properties: Option[Properties]): JsValue` for simplicity...

Reply:
That's what we do on the other responses, rely on the implicit. Not really sure how to do otherwise... And if do, should do a refactoring instead I think, as we should keep one way of doing for all responses. So will let it like that for now if you don't mind :)